multitenancy-spring-boot-autoconfigure/src/test/java/org/axonframework/extensions/multitenancy/autoconfig/MultiTenancyAutoConfigurationTest.java

```diff
@@ -124,6 +126,7 @@ void multiTenancyDisabled() {
                          assertThat(context).doesNotHaveBean(MultiTenantQueryUpdateEmitter.class);
                          assertThat(context).doesNotHaveBean(TenantEventSegmentFactory.class);
                          assertThat(context).doesNotHaveBean(MultiTenantDeadLetterQueueFactory.class);
+                         assertThat(context).doesNotHaveBean(MultiTenantEventScheduler.class);
                      });
```

Comment:
Why isn't this test validating the none-existening of the `TenantEventSchedulerSegmentFactory`?